subworkflows/local/igv_tracks.nf

```diff
@@ -0,0 +1,38 @@
+//
+// IGV TRACKS
+//
+
+include { JUNCTION_TRACK   } from '../../modules/local/junction_track'
+include { UCSC_WIGTOBIGWIG } from '../../modules/nf-core/ucsc/wigtobigwig/main'                                                                          
+
+workflow IGV_TRACKS {
+    take:
+        wig
+        path_sizes
+        spl_junc
+
+    main:
+        ch_versions = Channel.empty()
+
+        ch_wig = wig
+            .map{ meta, wigs ->
+            return[meta, wigs[1]] }
```

Comment:
Could you add a comment about why this is done. Something like "get the UniqueMultiple wig file"